abbott-smith.tei.xml

```diff
@@ -3694,7 +3694,7 @@ MM, <emph>VGT</emph>, s.v.): <ref osisRef="Acts.4.13">Ac 4:13</ref> (but cf. Tha
   <seg type="derivation">(&lt; <ref><foreign xml:lang="grc">ἀληθής</foreign></ref>),</seg>
   <seg type="septuagint">[in LXX for <foreign xml:lang="heb" n="H571">אֶמֶת</foreign>;]</seg>
 </etym>
-  <sense><gloss>true</gloss>, in the sense of real, ideal, genuine: <ref osisRef="Luke.16.11">Lk 16:11</ref> <ref osisRef="John.1.9">Jo 1:9</ref> <ref osisRef="John.4.23">4:23</ref>, <ref osisRef="John.4.37">37</ref> <ref osisRef="John.6.32">6:32</ref> <ref osisRef="John.7.28">7:28</ref> <ref osisRef="John.8.16">8:16</ref> <ref osisRef="John.15.1">15:1</ref> <ref osisRef="John.17.3">17:3</ref> <ref osisRef="John.19.35">19:35</ref>, <ref osisRef="1Thess.1.9">I Th 1:9</ref>, <ref osisRef="Heb.8.2">He 8:2</ref> <ref osisRef="Heb.9.24">9:24</ref> <ref osisRef="Heb.10.22">10:22</ref>, <ref osisRef="1John.2.8">I Jn 2:8</ref> <ref osisRef="1John.5.20">Jn 5:20</ref>, <ref osisRef="Rev.3.7">Re 3:7</ref>, <ref osisRef="Rev.3.14">14</ref> <ref osisRef="Rev.6.10">6:10</ref> <ref osisRef="Rev.15.3">15:3</ref> <ref osisRef="Rev.16.7">16:7</ref> <ref osisRef="Rev.19.2">19:2</ref>; = <foreign xml:lang="grc">ἀληθής</foreign>, <ref osisRef="Rev.19.9">Re 19:9</ref> <ref osisRef="Rev.21.5">21:5</ref> <ref osisRef="Rev.22.6">22:6</ref> (MM, <emph>VGT</emph>, s.v.).†</sense>

```

Comment:
Probably a different issue, but this markup is odd - normally, element content should be visible to the user, it seems to be using an element as an attribute:

```xml
<sense><gloss>true</gloss>
```

Better:

```xml
<sense gloss=true>
```

What exactly are the semantics?